src/modules/taskFunctions.js

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
Can you please remove this eslint-disable rule. so that linters will check each error if the file may have